sonar-text-plugin/src/main/resources/org/sonar/plugins/secrets/configuration/ms-teams.yaml

```diff
@@ -60,4 +60,4 @@ provider:
           # Structure:
           # https://tenant-name.webhook.office.com/webhookb2/guid@guid/IncomingWebhook/connectorId/guid
           # An MS GUID is in the following format XXXXXXXX-XXXX-XXXX-XXXX-XXXXXXXXXXXX where X is a hex digit
-          pattern: "((https://)?[a-z0-9_-]+\\.webhook\\.office\\.com/webhookb2/[a-z0-9\\-]+@[a-z0-9\\-]+/IncomingWebhook/[a-z0-9]+/[a-z0-9\\-]+)"
\ No newline at end of file
+          pattern: "((https://)?[a-z0-9_-]++\\.webhook\\.office\\.com/webhookb2/[a-z0-9\\-]++@[a-z0-9\\-]++/IncomingWebhook/[a-z0-9]++/[a-z0-9\\-]+)"
```

Comment:
Unfortunately, this will not fix the performance issue, I tested it on the `hashcat` project.
I even tested to change the last `+` to `++`, still no performance improvement.
Is it possible to specify how many characters are to be matched? Like:
`((https://)?[a-z0-9_-]{1, 50}\\.webhook\\.office\\.com/webhookb2/[a-z0-9\\-]{1, 50}@[a-z0-9\\-]{1, 50}/IncomingWebhook/[a-z0-9]{1, 50}/[a-z0-9\\-]{1, 50})`?